pages/syllabus.md

```diff
@@ -75,127 +126,142 @@ Class demonstrations, exercises, and homeworks will be done using [jupyter noteb
 
 # Course Modality
 
-This class is offered in person.  I'd say "face to face" but I'll probably wear a KN95 mask most of the time - I, and others in my immediate family, take extra care to avoid even minor colds and the like.  
```

Comment:
I like this explanation here but I'm not planning to wear a mask so took it out?